lib/server/createGraphQLPublication.js

```diff
@@ -9,14 +9,15 @@ import { contextToFunction } from './contextToFunction';
 
 // The 'pong' message is the only messages that is ignored by the client-side DDP parser
 const SUBSCRIPTION_MESSAGE_TYPE = 'pong';
+const { warn } = console;
 
 export function createGraphQLPublication({
   schema,
   context,
   publication = DEFAULT_PUBLICATION,
 } = {}) {
   if (!subscribe) {
-    console.warn('DDP-Apollo: You need graphl@0.11 or higher for subscription support');
+    warn('DDP-Apollo: You need graphl@0.11 or higher for subscription support');
```

Comment:
Noticed that Circle CI has an eslint warning about using `console.warn`. This is a cute trick I learned to get around it without have to use `eslint-disable` anywhere.

Reply:
I think I prefer the eslint-disable because it's more explicit. But I'm ok with this. It's better then the warning ;)